internal/aez/heater_regulators.go

```diff
@@ -33,6 +33,7 @@ type HTR struct {
 	HTR8OD uint16
 }
 
-func (s *HTR) read(buf io.Reader) error {
-	return binary.Read(buf, binary.BigEndian, s)
+// Read HTR
+func (htr *HTR) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, htr)
```

Comment:
how about a test of a real HTR buffer from a `.rac` file resulting in the expected HTR struct?